Public/Src/FrontEnd/Nuget/NugetResolver.cs

```diff
@@ -62,12 +69,89 @@ public override async Task<bool> InitResolverAsync(IResolverSettings resolverSet
 
             m_owningModules = new Dictionary<ModuleId, Package>();
 
-            foreach (var package in maybePackages.Result)
+            foreach (var package in maybePackages.Result.Values.SelectMany(v => v))
             {
                 m_packages[package.Id] = package;
                 m_owningModules[package.ModuleId] = package;
             }
 
+            if (Configuration.FrontEnd.GenerateCgManifestForNugets.IsValid ||
+                Configuration.FrontEnd.ValidateCgManifestForNugets.IsValid)
+            {
+                var cgManfiestGenerator = new NugetCgManifestGenerator(Context);
+                string generatedCgManifest = cgManfiestGenerator.GenerateCgManifestForPackages(maybePackages.Result);
+                string existingCgManifest = "INVALID";

```

Comment:
Why does this variable need to be declared here with a bogus value?   Please remove.

Reply:
ok, I'll add a null check inside the compare function